index.js

```diff
@@ -8,6 +8,8 @@ const ANDROID_ITEM_TYPE_IAP = 'inapp';
 const IOS_ITEM_TYPE_SUBSCRIPTION = 'sub';
 const IOS_ITEM_TYPE_IAP = 'iap';
 
+export const onPromotedProduct = 'iap-promoted-product';
```

Comment:
I think you can delete this one?

Reply:
Sure, but a well-defined constant is far better than a bare string at the point of use.

Reply:
Then I think the const variable should be renamed to something like `PROMOTED_PRODUCT'.